core/templates/dev/head/pages/topic_editor/questions/tag_misconception_modal_directive.html

```diff
@@ -0,0 +1,36 @@
+<div class="modal-header">
+  <h3>
+    Select the misconception to tag to:
+  </h3>
+</div>
+<div class="modal-body new-subtopic-title-editor">
+  Selected Misconception: <[selectedMisconception.getName()]>
+  <div ng-repeat="misconception in misconceptions track by $index">
+    <button class="btn btn-default" style="width: 100%" ng-click="selectMisconception(misconception)">
+      <[misconception.getName()]>
+    </button>
+    <div ng-if="selectedMisconception.getId() === misconception.getId()">
+      <label class="misconception-label"> Note to creators: </label>
+      <angular-html-bind html-data="misconception.getNotes()">
+      </angular-html-bind>
+      <label class="misconception-label"> Misconception Feedback: </label>
+      <angular-html-bind html-data="misconception.getFeedback()">
+      </angular-html-bind>
+    </div>
+  </div>
+  <span ng-click="toggleMisconceptionFeedbackUsage()" aria-hidden="true">
+    <i ng-if="!misconceptionFeedbackIsUsed" class="material-icons md-18">check_box_outline_blank</i>
+    <i ng-if="misconceptionFeedbackIsUsed" class="material-icons md-18">check_boxk</i>
```

Comment:
Typically we use the &...; codes for material-icons for better compatibility.

Is that "k" at the end of check_box supposed to be there?

Reply:
Done, changed it to that, and not, removed it.